benchmarks/votable.py

```diff
@@ -20,21 +17,33 @@
 id_data = np.arange(LARGE_SIZE, dtype=np.int64)
 flag_data = np.random.choice([True, False], LARGE_SIZE)
 quality_data = np.random.randint(0, 256, LARGE_SIZE, dtype=np.uint8)
+bool_data = np.random.randint(0, 2, LARGE_SIZE).astype(bool)
```

Comment:
Why is this code using [legacy random generation](https://numpy.org/doc/2.3/reference/random/generated/numpy.random.randint.html)? I am only highlighting it here, but it is a problem throughout.

Reply:
Shall I change this for all instances of this in `votable.py` for this PR, or shall I do so in a separate PR?